main/boost/APKBUILD

```diff
@@ -10,35 +10,53 @@ license="custom"
 depends=""
 depends_dev="linux-headers"
 makedepends="$depends_dev python2-dev>=2.6 python3-dev flex bison bzip2-dev zlib-dev"
-subpackages="$pkgname-dev $pkgname-doc"
+subpackages="$pkgname-doc"
 source="https://dl.bintray.com/boostorg/release/$pkgver/source/${pkgname}_$_pkgver.tar.bz2
 	boost-1.57.0-python-abi_letters.patch
 	boost-1.57.0-python-libpython_dep.patch
 	"
 builddir="$srcdir/${pkgname}_${_pkgver}"
 
-_libs="date_time
+_libs="
+	atomic
+	chrono
+	container
+	context
+	contract
+	coroutine
+	date_time
+	exception
+	fiber
 	filesystem
 	graph
 	iostreams
+	locale
+	log_setup
+	log
 	math
 	prg_exec_monitor
 	program_options
+	python2
 	python3
-	python
 	random
 	regex
 	serialization
 	signals
+	stacktrace_basic
+	stacktrace_noop
 	system
+	test_exec_monitor
 	thread
+	timer
+	type_erasure
 	unit_test_framework
 	wave
 	wserialization
 	"
 for _lib in $_libs; do
 	subpackages="$subpackages $pkgname-$_lib:_boostlib"
 done
+subpackages="$subpackages $pkgname-dev"
```

Comment:
Why do we move the `*.a` files to the runtime libs? As far I know those are only needed for static linking.

This commit also moves the `*.so` symlinks to the runtime libs, which breaks build of current packages that uses the shared libs, unless the libs are explicitly added to makedepends. I don't think we want to move there.

If this is needed, then the commit message should explain why.